### DISCUSSION TITLE
GBC alternative ?

### DISCUSSION THREAD

**Discussion by tattoo-girl** [haven1433/HexManiacAdvance#108] [Q&A] [answered]
Good day Haven, 

Is there an alternative tool to edit GBC ROMs  ?  Just as  HexManiacAdvance does?

I believe HexManiacAdvance only works for GBA ROMs.

I wonder if there's an alternative I could use to edit GBC ROMs in the same way

**Comment by haven1433** [comment, marked as answer]
Hello,

I'm not familiar with GBC tools, but I've heard that most people hack using the decomps and PokeCrystal. I'm not familiar with any tools similar to HMA for Gold/Silver/Crystal.

Good Luck!
